app/experimenter/experiments/models/nimbus.py

```diff
@@ -151,6 +151,7 @@ class NimbusExperiment(NimbusConstants, FilterMixin, models.Model):
         "NimbusFeatureConfig",
         blank=True,
     )
+    warn_on_feature_value_schema_invalid = models.BooleanField(default=False)
```

Comment:
That's a bit long for a field name, something shorter would probably be fine like `warn_schema` or something.

Reply:
I can change this, though FWIW I do tend to favor long descriptive names wherever possible